src/libprojectM/Common.hpp

```diff
@@ -26,215 +26,107 @@
 
 #ifndef COMMON_HPP
 #define COMMON_HPP
-#include <vector>
-#include <typeinfo>
-#include <cstdarg>
 #include <cassert>
+#include <cstdarg>
 #include <locale>
+#include <typeinfo>
+#include <vector>
 
-#ifdef _MSC_VER
-#define strcasecmp(s, t) _strcmpi(s, t)
-#endif
-
-#if defined(_MSC_VER ) && !defined(EYETUNE_WINRT)
-	#pragma warning( disable : 4244 4305 4996; once : 4018 )
-	#define WIN32_LEAN_AND_MEAN
-	#define NOMINMAX
-	#include <windows.h>
-	typedef unsigned int uint;
```

Comment:
Yeah, should've ideally used library-predefined types like uint32_t instead, if it was still being used.